main/admin/course_add.php

```diff
@@ -42,7 +42,14 @@
 $form->addElement('header', $tool_name);
 
 // Title
-$form->addText('title', get_lang('Title'), true);
+$form->addText(
+    'title',
+    get_lang('Title'),
+    true,
+    array(
+        'id' => 'title'
```

Comment:
I'm adding an `id` by default (if not is set) when a element is added with `FormValidator`. The `id` for elements will be `the name of form + underscore + the element name`. https://github.com/chamilo/chamilo-lms/commit/013347ff73167cca86fe9ffe3dd11a972bbfd8da

This should improve the use of the `for` attribute in the `label` tags (At least with the screen readers)